controllers/etcdadmconfig_controller.go

```diff
@@ -292,7 +288,7 @@ func (r *EtcdadmConfigReconciler) joinEtcd(ctx context.Context, scope *Scope) (_
 	log.Info("Machine Controller has set address on init machine")
 
 	etcdCerts := etcdCACertKeyPair()
-	rerr = etcdCerts.Lookup(
+	_ = etcdCerts.Lookup(
```

Comment:
I realize this code was swallowing the error earlier too, but if we encounter error here can we return it?

Reply:
done!